pages/desktop/frontend/versions.py

```diff
@@ -23,3 +30,94 @@ def versions_page_header(self):
     def latest_version_number(self):
         el = self.find_element(*self._latest_version_locator).text
         return el.split()[1].replace('Version ', '')
+
+    @property
+    def notice_message(self):
+        return self.find_elements(*self._notice_message_locator)[0]
```

Comment:
Using indices to locate an element is discouraged, especially in the format you are using here. Also, why `find_elements`?. Isn't there a single element on the page with `'.Card-contents .Notice-text'`?

Reply:
There are some cases where versions could have a warning message that can also be returned with that locator.
<img width="705" alt="Screenshot 2022-05-05 at 10 45 31" src="https://user-images.githubusercontent.com/95921687/166881957-68b52ece-755f-4262-91da-66008972b5a9.png">
I can change the locator to only find the message we want.